### PR TITLE
Add .npmrc to maestro-angular project

### PR DESCRIPTION
**Pull request by adiaaida**
This change adds the .npmrc file with the registry being https://dev.azure.com/dnceng/public/_artifacts/feed/dotnet-public-npm. This registry has an upstream to npmjs.

This addresses https://github.com/dotnet/arcade/issues/9565.

### CODE REVIEW

**Review by alexperovich — APPROVED** [review]

File: azure-pipelines.yml [hunk 1]
```diff
@@ -125,6 +125,11 @@ stages:
             solution: "**/*.sln"
             feedstoUse: config
 
+        # Authenticate for npmrc
+        - task: npmAuthenticate@0
```
Comment (riarenas) [inline azure-pipelines.yml:129]: We need to authenticate to public npm feeds? that's kind of annoying.

Comment (mmitche) [reply azure-pipelines.yml:129]: If we don't auth...then what happens? Does the build fail or does ingestion fail?

Comment (mmitche) [reply azure-pipelines.yml:129]: (and does this still work if building anonymously?)

Comment (adiaaida) [reply azure-pipelines.yml:129]: I originally tried without the auth in both the .npmrc and without the task to authenticate and it couldn't get to the packages in the feed. In what instances are we "building anonymously"? I will try to test it out

Comment (mmitche) [reply azure-pipelines.yml:129]: If I was a non-msft dev, does building locally work for me, assuming that the right ingestion has been done already?

Comment (adiaaida) [reply azure-pipelines.yml:129]: Let me do some testing

Comment (adiaaida) [reply azure-pipelines.yml:129]: (at the same time, this is arcade services, and specifically barviz. Do we care about that scenario?)

Comment (adiaaida) [reply azure-pipelines.yml:129]: I am getting "Resource not available for anonymous access" responses. I'll look at the settings of the feed

Comment (adiaaida) [reply azure-pipelines.yml:129]: Looks like the npmAuthenticate task isn't actually necessary. https://dev.azure.com/dnceng/internal/_build/results?buildId=1827407&view=logs&j=a5c6a902-5556-5a70-4b67-7f49bfe8ebe3&t=8955ed9e-1866-5780-bb6d-4a3931697224

Comment (adiaaida) [reply azure-pipelines.yml]: removed.

**Comment by riarenas** [comment]
@mmitche I see that our approvals didn't get reset after the new changes. What is the setting we need to enable in this repo for that?

**Review by riarenas — APPROVED** [review]

**Comment by riarenas** [comment]
>@mmitche I see that our approvals didn't get reset after the new changes. What is the setting we need to enable in this repo for that?

Forgot I'm an admin here. I made it look like Arcade's main branch policy.

**Review by mmitche — APPROVED** [review]

**Pull request merged**